src/content/keyAutoAdd/fynbos.ts

```diff
@@ -117,7 +113,7 @@ new KeyAutoAdd([
     maxDuration: LOGIN_WAIT_TIMEOUT,
   },
   { name: 'Finding wallet', run: findWallet },
-  { name: 'Finding form to add public key', run: findForm },
-  { name: 'Adding key', run: addKey },
+  { name: 'Get CSRF Token', run: getCSRFToken },
+  { name: 'Upload key', run: uploadKey },
```

Comment:
1. For consistency between steps and scripts
2. Using present continuous tense as we show these in progress directly.
```suggestion
  { name: 'Adding key', run: uploadKey },
```